test/specs/anInitialSuite.e2e.ts

```diff
@@ -34,6 +34,7 @@ describe('An Initial Suite', async () => {
   });
 
   step('Verify our extensions are not initially loaded', async () => {
+    utilities.pause(5)
```

Comment:
Added pause because of the flaky nature of test. Once the GHA PR is merged we can test if we need the pause.